charts/kubewarden-crds/templates/policyservers.yaml

```diff
@@ -172,6 +172,353 @@ spec:
                 description: Replicas is the number of desired replicas.
                 format: int32
                 type: integer
+              securityContexts:
```

Comment:
I wonder, why do we get all these `securityContexts` now?

Reply:
Ok, makes sense, added here:
https://github.com/kubewarden/kubewarden-controller/commit/e6ac7ba276f98b31e36671066573e2108ffd6d46
Resolving.